frontend/test/lit/test_device_api.py

```diff
@@ -28,10 +28,10 @@
 from catalyst.compiler import get_lib_path
 
 
-class DummyDevice(Device):
-    """A dummy device from the device API."""
+class NullDevice(Device):
```

Comment:
For these tests we do might want to create a custom class because we are testing the device api, in which case we might want to indicate this in the name though, `CustomDevice` instead of `NullDevice` for example.